### PR TITLE
Landing page mobile version

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/landing-page/landing-page-slide.tsx [hunk 1]
```diff
@@ -35,26 +31,42 @@ const Header = styled.div`
...
 const FlexContainer = styled.div`
   display: flex;
   justify-content: space-between;
   margin: 60px 0 40px 0;
-  @media (max-width: ${DESKTOP_WIDTH}px) {
+  @media (max-width: ${MOBILE_WIDTH}px) {
     display: block;
   }
 `;
+const FlexContainerReverse = styled.div`
```
Comment (EdmundMai) [inline src/components/landing-page/landing-page-slide.tsx]: name this something descriptive

File: src/components/landing-page/landing-page-slide.tsx [hunk 2]
```diff
@@ -35,26 +31,42 @@ const Header = styled.div`
...
+  display: flex;
+  justify-content: space-between;
+  margin: 60px 0 40px 0;
+  @media (max-width: ${MOBILE_WIDTH}px) {
+    flex-direction: column-reverse;
+  }
+`;
+
+const FlexWrapper = styled.div`
```
Comment (EdmundMai) [inline src/components/landing-page/landing-page-slide.tsx]: same

File: src/components/landing-page/landing-page-slide.tsx [hunk 3]
```diff
@@ -35,26 +31,42 @@ const Header = styled.div`
...
+  display: flex;
+  justify-content: center;
+  @media (max-width: ${MOBILE_WIDTH}px) {
+    display: grid;
+  }
+`;
 
 const Collum = styled.div`
```
Comment (EdmundMai) [inline src/components/landing-page/landing-page-slide.tsx]: Column

File: index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  if (req.hostname.match(/storage\./g)) {
+    res.sendFile(path.resolve("index.html"));
+  } else {
+    next();
+  }
+};
+
+app.get("/storage*", subdomainMiddleware, (req, res) => {
+  res.sendFile(path.resolve("index.html"));
+});
```
Comment (EdmundMai) [inline index.js]: what is this file for?

File: src/components/landing-page/landing-page-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: solved

File: src/components/landing-page/landing-page-slide.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: solved

File: src/components/landing-page/landing-page-slide.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: const Collum = styled.div`

File: index.js (hunk 4 above)
Comment (tenisakb) [reply index.js]: Deleted... all the stuff for routing I will move to another PR

**Pull request merged**